src/main/java/org/folio/rest/support/CollectionUtil.java

```diff
@@ -26,6 +26,15 @@ private CollectionUtil() {}
       .collect(Collectors.toList());
   }
 
+  public static <T> T getFirst(Collection<T> collection) {
+    if (collection == null) {
+      return null;
+    }
+
+    return collection.stream()
```

Comment:
what do you think to replace streams with iterator? it would be like:

`values!=null ? ofNullable(values.iterator().next()) : empty()`

Reply:
Make sense, changed.